cwms-data-api/src/main/java/cwms/cda/formatters/OutputFormatter.java

```diff
@@ -13,6 +13,10 @@ default <T extends CwmsDTOBase> T parseContent(String content, Class<T> type) {
         throw new UnsupportedOperationException("Unable to process your request. Deserialization of "
                 + getContentType() + " not yet supported.");
     }
+    default <T extends CwmsDTOBase> List<T> parseContentList(String content, Class<T> type) {
```

Comment:
It would seem "OutputFormatter" is no longer the correct name. (Not that we need to fix that right now.)

Reply:
Well it technically does take in an input and output a "formatted" DTO....